assets/interfaces.go

```diff
@@ -287,6 +287,22 @@ type Ticketer interface {
 	Type() string
 }
 
+// TopicUUID is the UUID of a topic
+type TopicUUID uuids.UUID
+
+// Topic categorizes tickets
+//
+//   {
+//     "uuid": "cd48bd11-08b9-44e3-9778-8e26adf08a7a",
+//     "name": "Weather"
+//   }
+//
+// @asset topic
+type Topic interface {
+	UUID() TopicUUID
+	Name() string
```

Comment:
we already have `MsgTopic` as a thing in the engine.. but can imagine one day it's just another type of topic